### PR TITLE
Gives androids the robotic speech span

### PR DESCRIPTION
**Pull request by XDTM**
:cl: XDTM
add: Androids now speak in a more robotic tone of voice.
/:cl:

### Reason for adding:
Makes the android experience more immersive, and also gives an opportunity to fool the crew into thinking that a borg is acting rogue.

### CODE REVIEW

File: code/modules/surgery/organs/organ_internal.dm [hunk 1]
```diff
@@ -661,6 +661,16 @@
 	name = "chattering bone \"tongue\""
 	chattering = TRUE
 
+/obj/item/organ/tongue/robot
+	name = "robotic voicebox"
+	desc = "A voice synthesizer that has can interface with organic lifeforms."
```
Comment (coiax) [inline code/modules/surgery/organs/organ_internal.dm]: that has can

Comment (XDTM) [inline code/modules/surgery/organs/organ_internal.dm]: i has can

**Comment by Cyberboss** [comment]
MY WAIFU SPEAKS PERFECT ENGRISH :-1:

jk

**Comment by XDTM** [comment]
I'll get a picture up when i can, but it's a voice analyzer with some wires anyway

**Pull request merged**